app/services/clean_unreceived_matches.rb

```diff
@@ -0,0 +1,14 @@
+class CleanUnreceivedMatches
+  def call
+    unreceived_matches = Match.missing
+    unreceived_matches.each do |u|
```

Comment:
This can be condensed into a single `each` call, right?

```ruby
Match.missing.each do |m|
  UnreceivedMatch.create(...) # instead of new + save
  m.delete
end
```

Reply:
:+1: